doc/main.html

```diff
@@ -491,6 +491,9 @@ <h4>Conformance section</h4>
 &lt;/section&gt;
 </pre>
         </div>
+
+        <p class="note">This section and additional text is not allowed for AG, OM, and EG docs.</p>
```

Comment:
```suggestion
        <p>This section is prohibited when the document is an administrative guideline, operations manual or engineering guideline.</p>
```

Reply:
Updated in https://github.com/SMPTE/html-pub/pull/287/commits/212bd78f67077d21a0bbed3745d62b46f3ada8ab